src/verb/internal_focus.rs

```diff
@@ -186,6 +194,10 @@ pub fn on_internal(
                 input_arg,
                 app_state,
             );
+            let tree_options = TreeOptions {
+                pattern: InputPattern::none(),
+                ..tree_options
+            };
```

Comment:
When `focus` is triggered by a verb, clear the input pattern. The idea is you're e.g. looking for a Git repo, so you type in some substring to get it selected, and then you trigger a `focus {git-repo}` verb which focuses you on the repo root and clears your filter pattern so you're ready to look around. But I think this is a change from the current behavior, so I'm not sure if you'll want this.

Reply:
It might be nicer to have a `clear_filter` option for verb definitions to control this behavior and then have some extra state in `CmdResult` for it.

Reply:
I know there's a UX problem there and I have in my TODO list to clarify this and implement something like this (I hadn't thought about a visible `clear_filter` option, this is actually a good idea).

Note: please don't attempt to code this clear_filter thing, I prefer for now to handle those core changes myself.

Reply:
OK, reverted this hunk.